app/javascript/components/items/QuantityController.js

```diff
@@ -0,0 +1,33 @@
+import React from "react"
+
+const QuantityController = (props) => {
+  const [state, setState] = React.useState({ item: props.item });
```

Comment:
This doesn't need to happen in this PR, but it would probably be a good idea to lift the state out of this Component into the Table as was done in `ItemRow`.

Reply:
Agreed and noted